Library/Homebrew/nightly-linux/centos7/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM centos:7
+
+# hadolint ignore=DL3008
```

Comment:
Can we get comments whenever we are ignoring linters. Also, is `hadolint` setup to validate these?